examples/preprocessing/plot_target_regressor_encoder.py

```diff
@@ -0,0 +1,121 @@
+"""
+=============================
+Target Encoder for Regressors
+=============================
+
+.. currentmodule:: sklearn.preprocessing
+
+The :class:`TargetRegressorEncoder` uses target statistics conditioned on
+the categorical features for encoding. In this example, we will compare
```

Comment:
```suggestion
The :class:`TargetRegressorEncoder` uses the value of the target to encode each
categorical feature. In this example, we will compare
```

This is a bit more generic than the current sentence, but I think if you don't already know how the target encoder works, you won't learn it from the current sentence. "target statistics conditioned on
the categorical features for encoding" is pretty dense, so I think a simpler sentence that gives the reader a vague idea of what is happening is nicer.